Dockerfile

```diff
@@ -5,7 +5,8 @@ FROM python:3.10-alpine
 RUN apk add --no-cache img --repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing
 
 # Setup other deps
-RUN apk add --no-cache git skopeo docker cargo
+RUN apk add --no-cache git skopeo docker cargo podman cni-plugins \
+    && sed -i 's/^#mount_program =/mount_program =/' /etc/containers/storage.conf
```

Comment:
why is the replacement necessary?

Reply:
This forces it to use `fuse-overlayfs` instead of the kernel module. I did forget to install `fuse-overlayfs` in the Dockerfile though...

```
/taskboot # podman pull alpine
Error: 'overlay' is not supported over overlayfs, a mount_program is required: backing file system is unsupported for this graph driver
/taskboot # sed -i 's/^#mount_program/mount_program/' /etc/containers/storage.conf
/taskboot # podman pull alpine
Error: overlay: can't stat program "/usr/bin/fuse-overlayfs": stat /usr/bin/fuse-overlayfs: no such file or directory
/taskboot # apk add fuse-overlayfs
...
/taskboot # podman pull alpine
Resolving "alpine" using unqualified-search registries (/etc/containers/registries.conf)
Trying to pull docker.io/library/alpine:latest...
Getting image source signatures
Copying blob c158987b0551 done
Copying config 49176f190c done
Writing manifest to image destination
Storing signatures
49176f190c7e9cdb51ac85ab6c6d5e4512352218190cd69b08e6fd803ffbf3da
/taskboot # 
```